### ISSUE TITLE
Set fixed stream player orientation when auto-rotate is disabled

### ISSUE THREAD

**Issue by sguinetti** [realOxy/M3UAndroid#38]
**Is your feature request related to a problem? Please describe.**
The video player should have the option to _set the orientation of the video when playing in full screen_. This feature would be useful in several situations.

**Describe the solution you'd like**
By default, the players follow the phone’s configuration for video orientation (portrait or landscape). However, for accessibility (which includes, but is not limited to TalkBack) or handling (in one hand) reasons, some users tend to disable auto-rotate and set the mobile to portrait orientation for avoid frustrating scenarios. This feature add the possibility to always play the live stream in a fixed orientation (landscape in several cases).

**Describe alternatives you've considered**
This is an optional feature for those who have disabled auto-rotate for accessibility or handling (or battery performance, as well) reasons and set the phone in portrait orientation. It does not affect those who use auto-rotate.

**Additional context**
This feature can be turn on or off from "Optional Player Features".

**Comment by realOxy** [comment]
Or can the app still rotate based on gravity even if the system's automatic rotation is disabled?

**Comment by sguinetti** [comment]
Let me rephrase, because I'm looking for the simplest way to implement it.

 In the optional features for the video player, the user goes to the option "switch between landscape and portrait modes". There are three possibilities, the user can disable (default), set to landscape or set portrait. 

Another suggestion is that a button can be added to switch from landscape to portrait and vice versa. But the idea is that the feature will work when auto-rotate is disabled.

**Comment by realOxy** [comment]
This feature has been implemented in commit #87ef0f9 and will appear in the next version. Before that, you can try to package your own application to test this new feature.

**Comment by sguinetti** [comment]
No problem. Thank you. Now, this is my turn to add more strings in Spanish language.

<string name="feat_setting_screen_rotating">rotación de pantalla</string>
<string name="feat_setting_screen_rotating_description">disponible cuando el mecanismo de rotación está desbloqueado</string>
